app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss

```diff
@@ -28,7 +28,12 @@
     }
 
     &::before {
-      @apply bg-gradient-to-t from-transparent via-transparent to-slate-800 bg-repeat bg-clip-border bottom-0 h-[20%] content-[''] left-0 absolute w-full opacity-80;
+      background-image: linear-gradient(
```

Comment:
Can we use the tailwind helper without writing a custom CSS here?

Reply:
@pranavrajs Updated the PR https://github.com/chatwoot/chatwoot/pull/7658/commits/1f5b3fb9b62a5bccd2eebe5e64bacae30ee9adff